hexrd/instrument.py

```diff
@@ -867,98 +868,56 @@ def extract_polar_maps(self, plane_data, imgser_dict,
         delta_eta = eta_edges[1] - eta_edges[0]
         ncols_eta = len(eta_edges) - 1
 
+        max_workers = os.cpu_count()
```

Comment:
@psavery -- this should be an input arg; the HEDM config object has a `multiprocessing` attribute that specifies this number.  I have found that on intel processors with hyperthreading, where the `cpu_count` is 2x the number of physical cores, asking for all of the cpus can lead to bad performance.  Now that is for multiprocessing instead of multithreading...  maybe it is ok to as for all available for the latter?